chart/kubeapps/Chart.yaml

```diff
@@ -33,4 +33,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 7.6.2-dev0
+version: 7.6.1-dev1
```

Comment:
Not sure if decreasing the version is intentional?

Reply:
Nope, must have missed the patch when resolving the conflict. Updated.